config/export_aws_resources_snapshots_sample.yml

```diff
@@ -0,0 +1,19 @@
+aws_profile: default
+region:
+      - us-east-1
```

Comment:
If currently you utility doesnt handle multiple region functionality .. then please make it as a string var instead of list.. please do the same in antman as well